servers/dsub/jobs/controllers/utils/extensions.py

```diff
@@ -23,4 +23,38 @@ def get_extensions(job):
         logs=logs.dsub_to_api(job),
         last_update=job.get('last-update'),
         envs=job['envs'],
-        source_file=script)
+        source_file=script,
+        events=hack_events())
+
+def hack_events():
```

Comment:
This was just for manual local testing - we don't want to merge this fake data into master. Please revert this part from the PR.

You can also leave a comment above
`# TODO: Plumb dstat events.`